src/index.js

```diff
@@ -0,0 +1,103 @@
+import './style.css';
+
+let ToDoArray = JSON.parse(localStorage.getItem('ToDoArray') || '[]');
```

Comment:
- [ ] As listed in the requirements there should be an array containing some tasks with each task having the following properties: description [`string`], completed [`bool`], index: [`number`]. Ensure you add this.
- [ ] Also ensure that this array of tasks is rendered dynamically to the DOM on page load.